app/views/groups/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render partial: 'shared/nav', locals: {title:'CATEGORIES', symbol:'', url: ''}%>
+
+<section class="body_container">
+  <p> <%= notice %></p>
+  <div  class="total_box"> 
+  <p class="group_name"> Total expenses for <%= current_user.name.titleize %> </p>
+   <p class="total_amount"> $<%=@total_expenses %> </p>
+  </div>
+  <div>
+  <% @groups.includes(:expenses).each do |group| %>
+  <%= render partial: 'shared/group', locals: { group: group } %>
+<%end%>
+      </div>
+      <div class="button_box">
+      <div class="push_box">
+      <%= button_to 'ADD CATEGORY', new_user_group_path(user_id: current_user.id), method: :get, class: 'add_category'%> </div>
```

Comment:
- [ ] The `ADD CATEGORY` button should be placed at the bottom of the page as seen in the original design. Please fix that 
### Your implentation
![image](https://github.com/Donmark2k/budget_app/assets/45460192/844bcbfd-34c7-4c2a-b8a6-64efbe2f2216)
### Original Design
![image](https://github.com/Donmark2k/budget_app/assets/45460192/3791e4a6-635d-4e87-92a3-7b78bef177bb)